src/vm/hints/uint256_utils.rs

```diff
@@ -27,16 +27,16 @@ pub fn uint256_add(
 ) -> Result<(), VirtualMachineError> {
     let shift: BigInt = bigint!(2).pow(128);
 
-    let a_addr = get_address_from_var_name("a", ids.clone(), vm, hint_ap_tracking)?;
-    let b_addr = get_address_from_var_name("b", ids.clone(), vm, hint_ap_tracking)?;
+    let a_maybe_rel = get_relocatable_from_var_name("a", ids.clone(), vm, hint_ap_tracking)?;
+    let b_maybe_rel = get_relocatable_from_var_name("b", ids.clone(), vm, hint_ap_tracking)?;
```

Comment:
It's not maybe.

Reply:
Sorry, forget about that.
[Done](https://github.com/lambdaclass/cleopatra_cairo/pull/304/commits/315739652f0711fe6e9fd700a8defb3253a56dd9)